### PR TITLE
13th Small Improvements

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Nerfed prestige and piety of Blackhand because he could invite other clans from the start. Gave more prestige and piety to my beloved Orgrim.
- Purged night elven orders of Cenarion Hold and Silverwing Sentinels.
- You can summon sayaadi via Summon Demon decision only if your public and true religion is Fel.
**_How to Test:_** `event WCCLS.610` to select a demon, there should be no sayaadi.
- Demons, spawned in the Voice in the Head event chain, no longer stay at your court if your public and true religion isn't Fel.
**_How to Test:_** `event WCFEL.9973` to summon this demon.
- C'thraxxi, spawned in the Ancient Creature event chain, no longer stay at your court if your true and public religion isn't Shath'yar.
**_How to Test:_** `event WCVOD.10015` to spawn c'thrax.
- Great War, Scourge Subjugation CBs give more score for battles and holding positions.
- The Burning Legion are limited to 3 attempts now.
**_How to Test:_** `event WCFEL.1000` to summon the Legion. After the 3rd summoning, the decision to summon the Legion in the Intrigue manu if you play Legion ruler shouldn't be allowed because the Legion is defeated.
- Rejuvenated Cho'gall, Gul'dan and Drek'thar.
- Generated Families rule is back. Added new default option: Infertile (only Infertile rulers generate families). Female rulers generate families as well as males now.
- Only rules with Alliance-related religion and culture can be invited to sail to Kalimdor with leader of Theramore.

Secret changlog:
- Renamed `highborne` from Quel'dorei to Highborne because there is already a culture with that name.

### CODE REVIEW

**Comment by MemeWizard** [comment]
> Nerfed prestige and piety of Blackhand because he could invite other clans from the start.

Problem not fixed

<details>
<summary>Click to expand</summary>

image
image

</details>


> 4-2
> 16-8

REEEEEEE

**Comment by zumbak04** [comment]
> > Nerfed prestige and piety of Blackhand because he could invite other clans from the start.
> 
> Problem not fixed

Problem nerfed!

> > 4-2
> > 16-8
> 
> REEEEEEE

REE!

File: events/game_rule_events.txt [hunk 1]
```diff
@@ -6739,1132 +6739,1289 @@ character_event = { # Delayed cleansing
...
+		mercenary = no
+    	holy_order = no
+    	is_landed = yes
+		OR = {
+			is_feudal = yes
+			is_nomadic = yes
+			is_tribal = yes
+		}
+		NOR = {
+			government = confucian_bureaucracy
```
Comment (MemeWizard) [inline events/game_rule_events.txt]: There is no confucian_bureaucracy in mod, but ok

Comment (zumbak04) [reply events/game_rule_events.txt]: Purged!

**Comment by MemeWizard** [comment]
> Nerfed prestige and piety of Blackhand because he could invite other clans from the start. Gave more prestige and piety to my beloved Orgrim.

AI immediately create titles and get prestige from it, so Blackhand still can invite other clans from the start.

<details>
<summary>Click to expand</summary>

image

</details>

**Comment by MemeWizard** [comment]
> The Burning Legion are limited to 3 attempts now.

There is no reason to show this decision if you never will be able to summon it after fall.

<details>
<summary>Click to expand</summary>

image

</details>

**Comment by MemeWizard** [comment]
> Generated Families rule is back. Added new default option: Infertile (only Infertile rulers generate families). Female rulers generate families as well as males now.

Generated children don't get education traits.

<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Moved adventure rebalance to separate PR: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/894 .

**Pull request merged**